rosidl_generator_py/package.xml

```diff
@@ -10,13 +10,27 @@
 
   <buildtool_export_depend>ament_cmake</buildtool_export_depend>
   <buildtool_export_depend>rosidl_cmake</buildtool_export_depend>
+  <buildtool_export_depend>rosidl_generator_c</buildtool_export_depend>
 
+  <build_depend>rosidl_generator_c</build_depend>
+  <!-- export??? -->
```

Comment:
In my opinion it doesn't need to export `rmw`. I did, however, clean up the dependencies 5fcb215.